src/symmetric_ops.jl

```diff
@@ -128,9 +131,40 @@ end
 @inline Base.(:*){dim, T}(v1::Vec{dim, T}, S1::SymmetricTensor{2, dim, T}) = dot(v1, S1)
 
 
+###########
+# Inverse #
```

Comment:
Moved symmetric inverse to `symmetric_ops.jl`. Will move back if this is not how we want it.